IssueTracker/IssueTrackerAPI/Controllers/ParticipantController.cs

```diff
@@ -59,12 +62,68 @@ public async Task<IResult> GetParticipant(int id)
             return Results.Ok(participantResponse);
 
         }
+        [HttpPost("createowner")]
+        [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme)]
+        public async Task<IResult> CreateOwner([FromBody]int projectId)
+        {
+            if (projectId > 0)
+            {
+                
+                var idclaim = User.Claims.FirstOrDefault(x => x.Type.Equals("UserId"));
```

Comment:
1. delete empty line
2. is it possible to have UserId claim empty/null in an Authorized method? if not, do not treat as nullable/empty